### ISSUE TITLE
Performance gap 

### ISSUE THREAD

**Issue by gbazad93** [Wangyixinxin/ACN#2]
Thanks for your contribution. I have read your paper and run your code for Brats 2018. Code runs fine but I  can not produce the paper results. For the ACN model with full modality the reported Dice scores on the paper are:

Whole Tumor = 89.22, Core Tumor = 85.18, Enhanced Tumor = 77.06 

While using the saved prediction results of the last epoch I have achieved the following performance, which has a significant difference. Could you please explain the reason? Thanks

Whole Tumor = 72, Core Tumor = 69, Enhanced Tumor = 66

**Comment by Wangyixinxin** [comment]
Hi, just in case, I retrained the code and got the similar results as in the paper.  You could check if you are using the right predictions, i.e. .batch*.pred.nii.gz and the right labels when you evaluate WT, TC, ET areas. Maybe you can share parts of your eval.log to see whether the inference process goes well.

**Comment by gbazad93** [comment]
Many thanks for your response. I do appreciate it. I have attached my evaluation code, which uses the generated files in the log folder. Could you please have a look? Thanks

[ACN_evluation.zip](https://github.com/Wangyixinxin/ACN/files/7014217/ACN_evluation.zip)

**Comment by Wangyixinxin** [comment]
Hi, it seems you are evaluating ed, net, et instead of wt, tc and et (sub-regions considered for evaluation).  wt is the sum of label 1,2,4 and tc is the sum of 2,4. Please refer to [brats18](https://www.med.upenn.edu/sbia/brats2018/tasks.html) for the evaluation labels. :)

**Comment by gbazad93** [comment]
Many thanks for your guidance.

**Comment by gbazad93** [comment]
Hi @Wangyixinxin,

I have retrained the model again with default setting and modified the evaluation code according to your explanation. The performance is still lower. Could you please have a look. I also loaded the trained weights and evaluated on the validation set but the performance was almost as follow one. Thanks 

![image](https://user-images.githubusercontent.com/48394338/130324523-20dfa691-f9d8-4409-81bb-98c2a32b9101.png)

**Comment by Wangyixinxin** [comment]
> Hi, it seems you are evaluating ed, net, et instead of wt, tc and et (sub-regions considered for evaluation). wt is the sum of label 1,2,4 and tc is the sum of 2,4. Please refer to [brats18](https://www.med.upenn.edu/sbia/brats2018/tasks.html) for the evaluation labels. :)

Hi, sorry that I made a typo. tc should be the sum of 1,4 and the dice score should be around 0.85. Please carefully check the link above and the right sub-regions should be et:4 tc:1,4 wt:1,2,4. 

<img width="532" alt="1629609494(1)" src="https://user-images.githubusercontent.com/54788216/130343348-83a86e11-7843-43d6-8100-a96354b9a652.png">

Besides, based on the correct evaluation, some small differences might be caused by the different data split.

**Comment by gbazad93** [comment]
Thanks for the information.
